zebra-rpc/src/methods.rs

```diff
@@ -48,15 +52,49 @@ pub trait Rpc {
     ///       note any other lightwalletd changes
     #[rpc(name = "getblockchaininfo")]
     fn get_blockchain_info(&self) -> Result<GetBlockChainInfo>;
+
+    /// getblock
+    ///
+    /// Returns requested block by height, encoded as hex.
+    ///
+    /// zcashd reference: <https://zcash.github.io/rpc/getblock.html>
+    ///
+    /// Result:
+    /// {
+    ///      "data": String, // The block encoded as hex
+    /// }
+    ///
+    /// Note: We only expose the `data` field lightwalletd uses the non-verbose
+    /// mode for all getblock calls: <https://github.com/zcash/lightwalletd/blob/v0.4.9/common/common.go#L232>
```

Comment:
Docs: `zcashd` supports a `hash` argument, but we don't need to.

```suggestion
    /// Note: We only expose the `data` field, because lightwalletd uses the non-verbose
    /// mode for all getblock calls: <https://github.com/zcash/lightwalletd/blob/v0.4.9/common/common.go#L232>
    ///
    /// `lightwalletd` only requests blocks by height, so we don't support getting blocks by hash.
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/3707/commits/2962449050a61bc998872b9d5693e01c5639387a